tests/suites/test_suite_pk.function

```diff
@@ -417,26 +417,30 @@ exit:
 void pk_rsa_overflow( )
 {
     mbedtls_pk_context pk;
-    size_t hash_len = SIZE_MAX;
+    size_t hash_len = SIZE_MAX, sig_len = SIZE_MAX;
+    unsigned char hash[50], sig[100];
 
     if( SIZE_MAX <= UINT_MAX )
         return;
 
+    memset( hash, 0x2a, sizeof hash );
```

Comment:
It is Mbed TLS style to use parenthesis with `sizeof`, even though it isn't strictly required by C.

See "Extra parentheses for return and sizeof" in the [Mbed TLS Style Guide](https://tls.mbed.org/kb/development/mbedtls-coding-standards).

Reply:
Consider adding a comment explaining why 0x2A is used for initializing the hash.

Reply:
These are preexisting issues, as are the hard-coded buffer sizes. Separate commits to fix these issues would be appreciated, but are outside the scope of the issue that triggered this PR.

Reply:
Which issues exactly are preexisting?

At least the `sizeof` issue is freshly added code and should use correct style, even if copy-pasted from a bad example.

Where does 0x2A come from if this is preexisting? It also seems to be freshly added code.

Reply:
OK. Unfortunate to have those issues, but I am fine with these as they are consistent with the rest of the file.

Reply:
As @gilles-peskine-arm stated, 0x2a and coding style issues are consistent with the rest of the file.